Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -45,13 +45,17 @@ public void MutateSeed(SeedData seed, float severity)
         // Kill the plant (30)
         MutateBool(ref seed.Viable         , false , 30 , totalbits , severity);
 
-        // Fun (70)
+        // Fun (90)
         MutateBool(ref seed.Seedless       , true  , 10 , totalbits , severity);
         MutateBool(ref seed.Slip           , true  , 10 , totalbits , severity);
         MutateBool(ref seed.Sentient       , true  , 10 , totalbits , severity);
         MutateBool(ref seed.Ligneous       , true  , 10 , totalbits , severity);
         MutateBool(ref seed.Bioluminescent , true  , 10 , totalbits , severity);
+        MutateBool(ref seed.TurnIntoKudzu , true  , 10 , totalbits , severity);
+        MutateBool(ref seed.Screaming , true  , 10 , totalbits , severity);
         seed.BioluminescentColor = RandomColor(seed.BioluminescentColor, 10, totalbits, severity);
+        // ConstantUpgade (10)
```

Comment:
Wouldn't "harvest type" here be better? But if you rename `MutateUpgrade` to `MutateHarvestType`, that would also be obvious without a comment.

Reply:
> Of course, congratulations and good work on your first PR :)
Thank you for review Partmedia !
i will further increase my coding skills , and i mostly plan to work with botany.
there are still a lot of botany features that can be implemented! dont go easy on me, expirience here can help me with future job.
english is not my primary language, also sorry about that.

Reply:
> > Of course, congratulations and good work on your first PR :)


Thank you for review Partmedia !
 i will further increase my coding skills , and i mostly plan to work with botany.
 there are still a lot of botany features that can be implemented! dont go easy on me, expirience here can help me with future job.
 english is not my primary language, also sorry about that.